app/src/main/res/values/strings.xml

```diff
@@ -63,4 +63,14 @@
     <string name="title_cancel">Cancel</string>
     <string name="title_waiting">Waiting</string>
     <string name="title_no_booking">You have no booking yet</string>
+    <string name="title_booking_id">Booking ID</string>
+    <string name="action_search">Search</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.action_search` appears to be unused
 Unused resources make applications larger and slow down builds.